### DISCUSSION TITLE
(use case) git-bug for bug-bounty?

### DISCUSSION THREAD

**Discussion by ghost** [git-bug/git-bug#953] [Q&A] [answered]
Hi all.

I would like to know what you think about the idea of git-bug being used as a bug-bounty... my idea is to provide a decentralized, distributed way of bugs and who solves it gets some reward... I want to know what you all think about this idea.

*What you all think about this idea?*

**Comment by MichaelMure** [comment, marked as answer]
Personally I don't care so much about what people would use git-bug for, as long as they can use it for that task. So ... what feature would be needed to do what you are talking about?

If that's a bounty payment system, I'd think that would be out of scope for git-bug, but that's the beauty of open-source, you certainly can integrate git-bug into something bigger or even fork the project to make it work for you.

**Reply by ghost** [reply]
Hi MichaelMure, thank you for feedback.
